constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -102,6 +104,13 @@ export class ClassEditPage extends React.Component {
         this.setState({ [value]: event.target.value });
     };
 
+    onChangeFullState = (e) => {
+        const value = e.target.value;
+        this.setState({
+            fullState: value,
```

Comment:
e.target.value returns a string, So we'll need to use `parseInt()` to convert this to an integer